src/context/fetch.test.ts

```diff
@@ -34,3 +36,26 @@ test('aguments RequestInit with the Record<string, string> headers', () => {
   expect(headers.get('x-msw-bypass')).toEqual('true')
   expect(headers.get('authorization')).toEqual('token')
 })
+
+test('creates new FormData from the plain object body if it is multipart/form-data', () => {
+  // User sends FormData in their app:
+  // const formData = new FormData();
+  // formData.append("key", "value");
+  // fetch("/api", { method: "POST", body: formData });
+  // User decides to proxy the intercepted request to real endpoint in their handler.js:
+  // const proxy = await ctx.fetch(req);
+  // Now fetch() from fetch.ts will receive the body as a plain object even though the user send FormData:
+  const input = createMockedRequest({
+    method: 'POST',
+    body: {
+      key: 'value',
+    },
+    headers: new HeadersPolyfill({
+      'content-type':
+        'multipart/form-data; boundary=----WebKitFormBoundaryostedjdEBW2xxin5',
```

Comment:
A useful warning from MDN:

> Warning: When using FormData to submit POST requests using [XMLHttpRequest](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest) or the [Fetch_API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) with the multipart/form-data [Content-Type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) (e.g. when uploading Files and Blobs to the server), do not explicitly set the Content-Type header on the request. Doing so will prevent the browser from being able to set the Content-Type header with the boundary expression it will use to delimit form fields in the request body.
> —[Source](https://developer.mozilla.org/en-US/docs/Web/API/FormData/Using_FormData_Objects)

So we do have to remove the `Content-Type` request header for the `FormData` to be handled correctly.